.github/workflows/create_release.yml

```
@@ -3,11 +3,53 @@ name: Create release
+      - CI
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v5.0.0
+
+    - name: Twingate Setup
+      uses: twingate/github-action@V1.1
+      with:
+        service-key: ${{ secrets.TWINGATE_SERVICE_ACCOUNT }}
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)